src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,32 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, callbackTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={classNames(
+              { 'is-active': selectedTab.id === tab.id },
+            )}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => callbackTabSelected(tab)}
```

Comment:
```suggestion
              onClick={() => {
                if(tab changed) {
                  onTabSelected(tab)
                }
              }}
```
![image](https://github.com/mate-academy/react_tabs-js/assets/55200530/47d0d4b4-347d-41c6-8224-36a86c46b2f7)